raijin_scripts/deploy/dea/datacube-ensure-user.py

```diff
@@ -83,7 +79,7 @@ def find_credentials(pgpass, host, dbcreds):
     else:
         with pgpass.open() as src:
             for line in src:
-                if line.strip():
+                if line.partition('#')[0].strip():
```

Comment:
This line works, but it isn't very readable.

Something like `line.strip().startswith('#')` is probably nicer.

Reply:
Updated the line to `not line.startswith('#') and line.strip()`

Reply:
Just in case a comment has spaces before it, I've made it `not line.strip().startswith('#')`.